cmd/registry/cmd/get.go

```diff
@@ -61,7 +61,11 @@ var getCmd = &cobra.Command{
 				_, err = core.GetSpec(ctx, client, m, getContents, core.PrintSpecDetail)
 			}
 		} else if m := names.ArtifactRegexp().FindStringSubmatch(name); m != nil {
-			_, err = core.GetArtifact(ctx, client, m, true, core.PrintArtifactDetail)
+			if getContents {
+				_, err = core.GetArtifact(ctx, client, m, getContents, core.PrintArtifactContents)
+			} else {
+				_, err = core.GetArtifact(ctx, client, m, getContents, core.PrintArtifactDetail)
+			}
```

Comment:
This lets "registry get" display both the Artifact message and its contents, depending on whether the --contents flag is included.